tests/unit/pynwb_tests/test_core.py

```diff
@@ -206,8 +206,9 @@ def test_extra_columns(self):
     def test_indexed_dynamic_table_region(self):
         table = self.with_columns_and_data()
 
-        dynamic_table_region = DynamicTableRegion('dtr', [0, 1], 'desc', table=table)
-        assert dynamic_table_region[slice(0, 1)]
+        dynamic_table_region = DynamicTableRegion('dtr', [0, 1, 1], 'desc', table=table)
+        fetch_ids = [x[1] for x in dynamic_table_region[:3]]
+        assert fetch_ids == [1, 2, 2]
```

Comment:
Is there a reason you're not using [assertListEqual](https://docs.python.org/3.7/library/unittest.html#unittest.TestCase.assertListEqual) or [assert_array_equal](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.testing.assert_array_equal.html)?

Reply:
no, I'm happy to use those instead. What is the advantage?

Reply:
ah ok that makes sense